dockerutil/image.go

```diff
@@ -22,6 +23,28 @@ import (
 
 const diskFullStorageDriver = "vfs"
 
+var usrLibMultiarchDir = map[string]string{
+	"arm64": "/usr/lib/aarch64-linux-gnu",
+	"amd64": "/usr/lib/x86_64-linux-gnu",
+}
+
+// Adapted from https://github.com/NVIDIA/libnvidia-container/blob/v1.15.0/src/nvc_container.c#L152-L165
+var UsrLibDirs = map[string]string{
+	// Debian-based distros use a multi-arch directory.
+	"debian": usrLibMultiarchDir[goruntime.GOARCH],
```

Comment:
This _could_ be blank, which might make for an interesting experience.
You'll likely need to account for this in `UsrLibDir()`.

Reply:
@dannykopping would you say it's panic-worthy?

Reply:
I don't know enough about the context in which this is running to give you an informed opinion, I'm afraid.

Reply:
I'll see if I can make it a build-time error instead.

Reply:
Sysbox is only offered for amd64 and arm64 so someone would have to be compiling their own envbox for no reason to get this to happen

If you want to prevent people from building envbox themselves for other arches that's fine but it seems unlikely that anyone will try anyways

Reply:
Better off defining `usrLibMultiarchDir` separately via implicit build constraints (e.g. `image_linux_arm64.go`)